src/main/groovy/org/assertj/generator/gradle/internal/tasks/config/DefaultAssertJGeneratorOptions.groovy

```diff
@@ -37,43 +37,24 @@ class DefaultAssertJGeneratorOptions implements AssertJGeneratorOptions, Seriali
     private Templates templates
     protected EntryPointGeneratorOptions _entryPoints
 
-    protected String outputDir
+    final DirectoryProperty outputDir
```

Comment:
This change was required to remove the custom source set and properly handle when it changes.